src/kontrol/foundry.py

```diff
@@ -550,7 +553,66 @@ def foundry_show(
             res_lines += print_failure_info(proof, kcfg_explore, counterexample_info)
             res_lines += Foundry.help_info()
 
-    return '\n'.join(res_lines)
+    if to_kevm_claims:
+        _foundry_labels = [
+            prod.label
+            for prod in foundry.kevm.definition.all_modules_dict['FOUNDRY-CHEAT-CODES'].productions
+            if prod.label is not None
+        ]
```

Comment:
Maybe define this as a property of the `Foundry` class? But it's fine either way for now, since it is used only here.

Reply:
Will wait for that if it's needed later.